cluster-scope/overlays/moc/curator/oauths/moc-sso-client-secret.enc.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: v1
+kind: Secret
+metadata:
```

Comment:
@larsks can you add the following annotations, as found [here](https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/moc/infra/oauths/moc-sso-client-secret.enc.yaml#L6) 

```
    annotations:
        argocd.argoproj.io/compare-options: IgnoreExtraneous
        argocd.argoproj.io/sync-options: Prune=false
```

See [here](https://github.com/operate-first/apps/issues/278) for context.

Reply:
Done.